lib/coinbase/client.rb

```diff
@@ -6,7 +6,7 @@
 The version of the OpenAPI document: 0.0.1-alpha
 
 Generated by: https://openapi-generator.tech
-Generator version: 7.7.0
+Generator version: 7.8.0
```

Comment:
@John-peterson-coinbase i have updated clients in my PR here - https://github.com/coinbase/coinbase-sdk-nodejs/pull/233